src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -315,3 +317,22 @@ def prepare_logs_argument(c):
                   'spring-cloud application-configuration-service git repo remove']:
         with self.argument_context(scope) as c:
             c.argument('name', help="Required unique name to label each item of git configs.")
+
+    for scope in ['spring-cloud build-service buildpacks-binding create',
+                  'spring-cloud build-service buildpacks-binding set']:
+        with self.argument_context(scope) as c:
+            c.argument('type', arg_type=get_enum_type(BuildpacksBindingType), help='Required type for buildpacks binding.')
```

Comment:
Use SDK value? you don't need to build another same wheel

Reply:
Thanks, not knowing that SDK provide the same enum